src/main/java/nl/igorski/lib/audio/MWEngine.java

```diff
@@ -139,7 +139,7 @@ public void createOutput( int aSampleRate, int aBufferSize, int aOutputChannels
         if ( Build.FINGERPRINT.startsWith( "generic" ))
             SAMPLE_RATE = 8000;
 
-        MWEngineCore.setup( BUFFER_SIZE, SAMPLE_RATE, OUTPUT_CHANNELS );
+        AudioEngine.setup( BUFFER_SIZE, SAMPLE_RATE, OUTPUT_CHANNELS );
 
```

Comment:
Updating MWEngine class to use newly exposed _AudioEngine_ directly. The public API of the MWEngine class remains the same for backwards compatibility.